java/lib/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    // Apply the kotlin-jvm plugin for Kotlin JVM projects.
+    kotlin("jvm") version "1.6.21"
+    // Apply the maven-publish plugin for publishing the library.
+    id("maven-publish")
+}
+
+group = "io.amplica.frequency.graph-sdk"
```

Comment:
I think our group would be `io.amplica.graphsdk` based on our current java packages.